docs/spelling_wordlist.txt

```diff
@@ -17,7 +17,6 @@ dataset
 datasets
 Deis
 devel
-DigitalOcean
```

Comment:
We still mention "DigitalOcean" (now unlinked) in whatsnew.rst, so this probably ought to stay.